agent/lib/kontena/workers/node_stats_worker.rb

```diff
@@ -116,13 +125,15 @@ def publish_node_stats
         network: network_traffic,
         time: Time.now.utc.to_s
       }
-      rpc_client.async.notification('/nodes/stats', [data])
-      send_statsd_metrics(data)
+    end
+
+    # @param [Hash] data
+    def send_node_stats(data)
+      rpc_client.async.notification('/nodes/stats', [@node.id, data])
     end
 
     # @param [Hash] event
     def send_statsd_metrics(event)
-      return unless statsd
       key_base = "#{docker_info['Name']}"
```

Comment:
Docker root dir is best guess here.. you can open an issue about this if you think it's needed.